cinema/management/commands/wait_for_db.py

```
@@ -0,0 +1,22 @@
+import time
+from django.core.management.base import BaseCommand, CommandError
+
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    """The command to wait for db connections"""
+
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        db_connection = None
+        while not db_connection:
+            try:
+                db_connection = connections["default"]
+                db_connection.cursor()
+            except OperationalError:
+                self.stdout.write("Database unavailable, waiting 1 second...")
+                time.sleep(1)
```

The loop retries indefinitely on OperationalError which satisfies the requirement, but you may want to consider:
- Adding an optional max retry count or overall timeout so the command fails fast in CI when DB settings are wrong.
- Implementing an exponential backoff or making the sleep duration configurable via an env var or command option to ease debugging and avoid tight retry loops.
These are suggestions to improve robustness; current behavior meets the task but may make debugging long startup failures harder.